src/mdx-pages/guides/options.mdx

```diff
@@ -193,6 +193,28 @@ See the file `sandbox/responsive.html.example` for an example of a responsive pl
 
 This option is inherited from the [`Component` base class](#component-options).
 
+### `disablePictureInPicture`
+
+> Type: `boolean`
+
+If `true`, switching the video element into picture-in-picture is disabled. Defautl is `false`.
+
+This has no effect on Firefox, which implements a proprietary picture-in-picture mode which does not implement the standard picture-in-picture API.
+
+This does not disable the document picture-in-picture mode which allows the player element to put into a PiP window.
+
+### `enableDocumentPictureInPicture`
+
+> Type: `boolean`
+
+_Since 8.3.0_
+
+If `true`, the [documentPictureInPicture API](https://developer.chrome.com/docs/web-platform/document-picture-in-picture/) will be used for picture in picture, if available. Defaults to `false`, but may default to `true` when the feature has become established.
```

Comment:
nit: I think we want to include hyphens in "picture-in-picture", just to keep the doc consistent.